Batch visibility zandleem (#51)
inbo/mbag: src/markdown/mas/steekproefontwerp_zandleemstreek.Rmd

  mapview(st_crop(exclusie_punten, detail), col.regions = "black", legend = FALSE)
```

### Exclusie op basis van zichtbaarheid

> Berekening cumulatieve zichtbaarheidsindex binnen straal van 25 m rond telpunt (zone bewegingsvrijheid van de waarnemer) in een raster (resolutie 5 m) van locaties die 10 m van elkaar liggen.

Reviewer: ```suggestion
Berekening cumulatieve zichtbaarheidsindex waarbij de waarnemer een bewegingsvrijheid heeft van 25 m rondom het telpunt.
De zichtbaarheidsindex wordt berekend vanaf een set van locaties die 10 m van elkaar liggen in deze zone van bewegingsvrijheid en geeft de proportie aan van zichtbare rastercellen in de telcirkel met 300 m straal.
De cumulatieve zichtbaarheidsindex is dan de proportie die zichtbaar is vanuit minstens één locatie binnen de zone van bewegingsvrijheid.
```

Door middel van Digital Surface Model (DSM) en Digital Terrain Model (DTM).
Aanpassing aan **GVI** package.
Zie: `visibility.Rmd`.

In de pipeline werd gebruik gemaakt van batching per 200 telpunten om rekentijd in te perken.
Zie:\
<https://docs.ropensci.org/tarchetypes/reference/tar_group_size.html>\
<https://books.ropensci.org/targets/dynamic.html#performance-and-batching>

> We verwijderen telpunten waar de cumulatieve 'viewshed of visibility' (cvvi) kleiner is dan 0.10.

Reviewer: ```suggestion
We verwijderen telpunten waar de cumulatieve 'viewshed visibility index' (cvvi) kleiner is dan 0.10.
```
Eventueel eens in aparte Rmd een aantal voorbeeldkaartjes (orthofoto achtergrond) maken waarin de cumulatieve viewshed gevisualiseerd wordt (via output_type = "summed viewshed") voor gevallen waar de cvvi ongeveer gelijk is aan 0.05, 0.1, 0.15, 0.2, 0.3, ...?

***Hoe is het nu gedaan?***

Target sample size is het maximaal aantal plots die binnen de OL, HOL perimeter kunnen gelegd worden.

```{r}
st_area(selectie_openheid_klasses) / units::set_units(300 * 300 * pi, m^2)
```

Minima onder x4 scenario.
4 strata (OL, HOL en binnen, buiten sbp) met minstens 50 telpunten per stratum.
x4 betekend halvering betrouwbaarheidsinterval --> minimum sample size is 200 telpunten per stratum.

> Niet echt belangrijk hier omdat we een maximum steekproefkader trekken en de tellers gewoon zoveel mogelijk punten laten doen (volgnummer).

Reviewer: ```suggestion
Niet echt belangrijk hier omdat we een maximum steekproef trekken en de tellers gewoon zoveel mogelijk punten laten doen (volgnummer).
```
